google-ads/pom.xml

```diff
@@ -44,9 +46,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>com.google.api</groupId>
-      <artifactId>gax</artifactId>
-      <version>${gax.version}</version>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>26.0-android</version>
```

Comment:
Is usage of the `-android` version intentional? If so, please add a comment explaining why.

Reply:
Yes, this is for JDK7 compatibility and is used by GAX etc. Added comments.